ci/scripts/quick-bazel.sh

```diff
@@ -8,7 +8,82 @@
 # This doesn't install dependencies, but should otherwise behave the
 # same as what CI would do on a pull request
 
-bazel query "rdeps(//..., //hw:verilator)" | \
-            sed -e 's/^/-/' | \
-            xargs bazel test --nobuild_tests_only --test_tag_filters=-cw310,-verilator -- //...
+set -e -x
+
+DEFAULT="master"
+
+git config advice.detachedHead false
+
+ORIGINAL_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD 2>/dev/null || echo "")
```

Comment:
More common is probably `|| true` here

Reply:
if we're running at a detached head, then `git switch "$ORIGINAL_BRANCH_NAME"` will fail and use the SHA instead to restore the originally checked out commit and the source tree in case CI depends on it.